web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -140,26 +140,24 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
       },
       approvalRequired: true,
       image: userUtils.defaultAvatar('champion'),
-      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      text: $translate.instant('Youth Champion Text...'),
```

Comment:
@daniel : maybe we should have a "catchphrase " service which contians this one and mines from my PR for rewording of join a dojo